<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastStrengthenDlg</class>
 <widget class="QDialog" name="ContrastStrengthenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>700</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>17</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBoxCoordinate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>331</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>坐标微调</string>
     </property>
     <widget class="QLabel" name="txtPointSerial">
      <property name="geometry">
       <rect>
        <x>87</x>
        <y>37</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>点1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="txtPoint1Coor">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(100,100)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveUp">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>80</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveDown">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>180</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveRight">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>130</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMoveLeft">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>130</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>节点</string>
     </property>
     <widget class="QRadioButton" name="radioButton_two">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>34</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_four">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>34</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_eight">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>94</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_sixteen">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>94</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBoxBin">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>点选择</string>
     </property>
     <widget class="QPushButton" name="btnSupPointSelect">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>90</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddPointSelect">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>90</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderPointSelect">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="txtPointSelect">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxBin_2">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>210</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>点坐标Y值调整</string>
     </property>
     <widget class="QPushButton" name="btnSubPointY">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>84</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddPointY">
      <property name="geometry">
       <rect>
        <x>294</x>
        <y>84</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderPointY">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>84</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="txtPointY">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>64</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>426</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>模式选择</string>
   </property>
   <widget class="QRadioButton" name="radioButtonAdvanced">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>高级</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBasic">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>基础</string>
    </property>
   </widget>
  </widget>
  <widget class="ContrastStrengthenAdjustLabel" name="labelPicList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>700</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContrastStrengthenAdjustLabel</class>
   <extends>QLabel</extends>
   <header>ImgpreprocessModule/Roi/ContrastStrengthenAdjustLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
